internal/rbd/rbd_attach_test.go

```diff
@@ -104,3 +106,62 @@ func TestParseMapOptions(t *testing.T) {
 		})
 	}
 }
+
+func Test_appendCrushLocationMapOptions(t *testing.T) {
+	t.Parallel()
+	type input struct {
+		mapOptions       string
+		crushLocationMap map[string]string
+	}
+	tests := []struct {
+		name string
+		args input
+		want string
+	}{
+		{
+			name: "both empty mapOptions and crushLocationMap",
+			args: input{
+				mapOptions:       "",
+				crushLocationMap: nil,
+			},
+			want: "",
+		},
+		{
+			name: "empty mapOptions and filled crushLocationMap",
+			args: input{
+				mapOptions: "",
+				crushLocationMap: map[string]string{
+					"region": "west",
+				},
+			},
+			want: "read_from_replica=localize,crush_location=region:west",
+		},
+		{
+			name: "filled mapOptions and crushLocationMap",
+			args: input{
+				mapOptions: "notrim=true",
```

Comment:
`notrim` is just a flag, it doesn't take any arguments:
```suggestion
				mapOptions: "notrim",
```